endpoint/endpoint.go

```diff
@@ -0,0 +1,6 @@
+package endpoint
+
+type Endpoint struct {
```

Comment:
We should discuss this. I think we need something that captures the dns record on a very detailed level (e.g. per record TTL, type: A, etc.) but also on a higher level (e.g. serviceName -> IPs).